x-pack/plugins/cloud_security_posture/public/common/navigation/constants.ts

```diff
@@ -12,10 +12,15 @@ import type { CspPage, CspNavigationItem } from './types';
 export const allNavigationItems: Record<CspPage, CspNavigationItem> = {
   dashboard: { name: TEXT.DASHBOARD, path: '/dashboard' },
   findings: { name: TEXT.FINDINGS, path: '/findings' },
-  rules: { name: 'Rules', path: '/rules', disabled: !INTERNAL_FEATURE_FLAGS.showBenchmarks },
+  rules: {
+    name: 'Rules',
+    path: '/benchmark/:packageId/:policyId/rules',
+    disabled: !INTERNAL_FEATURE_FLAGS.showBenchmarks,
+  },
```

Comment:
this shows up on the side navigation although it shouldn't

tracked at:
- https://github.com/elastic/security-team/issues/3258